tests/containers/podman_netavark.pm

```diff
@@ -93,11 +95,11 @@ sub run {
         return 1;
     }
 
+    chk_install_netavark_pkg;
     if (is_cni_default || is_cni_in_tw) {
-        switch_to_netavark;
+        switch_to_netawark;
```

Comment:
I used an incorrect name `netawark`, but not affecting the code syntax. I will update it in next commits.